index.html

```diff
@@ -33,133 +33,18 @@ <h1>
       don’t hesitate to contact me.
     </p>
     <section class="social">
-      <img src="images/twitter.png" alt="">
-      <img src="images/Linkedin icon.png" alt="">
-      <img src="images/medium.png" alt="">
-      <img src="images/github.png" alt="">
-      <img src="images/angellist.png" alt="">
+      <ul>
+        <li><img src="images/twitter.png" alt=""></li>
+        <li><img src="images/Linkedin icon.png" alt=""></li>
+        <li><img src="images/medium.png" alt=""></li>
+        <li><img src="images/github.png" alt=""></li>
```

Comment:
- [Optional] Good job wrapping your icons in a `ul` tag @mzirkof, But Your icons are supposed to be a link to your pages in each of them. as such you should wrap them inside an `a` tag. 
> Something like...
```html
<li><a href="your link here..."><img src="images/twitter.png" alt=""></a></li>
```